index.html

```diff
@@ -14,14 +14,20 @@
     <head>
         <nav>
             <a src="" id="logo">My Logo</a>
+            <ul class="menu-items">
+                <li><a href="#workspace">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
             <img src="./images/Enabled.svg" alt="Menu" id="menu">
         </nav>
     </head>
     <main>
```

Comment:
Greeting!

Great work so far! But you have a few things to change before moving to the next steps.

- Kindly make sure you stick to desgin as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma Design
>
><img width="219" alt="Screenshot 2023-06-07 at 5 53 25 PM" src="https://github.com/SnakyMz/Portfolio/assets/91237525/e423e8dc-1cb0-4900-bfdf-cc9657c8e023">
>
> ## Your Design
>
><img width="1391" alt="Screenshot 2023-06-07 at 5 44 59 PM" src="https://github.com/SnakyMz/Portfolio/assets/91237525/0c0884f9-254d-41f0-8924-18ad8d84bc60">
><img width="1397" alt="Screenshot 2023-06-07 at 5 45 16 PM" src="https://github.com/SnakyMz/Portfolio/assets/91237525/70a06d29-7b31-4dd4-90fa-921ebb1c23e0">
><img width="1398" alt="Screenshot 2023-06-07 at 5 45 25 PM" src="https://github.com/SnakyMz/Portfolio/assets/91237525/35d4071d-bd4a-4d4f-8012-506cce34d067">
><img width="1394" alt="Screenshot 2023-06-07 at 5 45 37 PM" src="https://github.com/SnakyMz/Portfolio/assets/91237525/4700e666-09fc-4fdb-8e8d-a1f4f209a405">